server/controllers/storage/index.js

```diff
@@ -0,0 +1,14 @@
+const { getSignedUrl } = require('../../helpers/storageHelpers');
```

Comment:
this should be **generateV4SignedUrl** instead of getSignedUrl and also the same on line 7
```suggestion
const { generateV4SignedUrl } = require('../../helpers/storageHelpers');
```